cluster/gce/manifests/glbc.manifest

```diff
@@ -15,7 +15,7 @@ spec:
   terminationGracePeriodSeconds: 600
   hostNetwork: true
   containers:
-  - image: gcr.io/k8s-ingress-image-push/ingress-gce-glbc-amd64:v1.12.0
+  - image: gcr.io/k8s-ingress-image-push/ingress-gce-glbc-amd64:nightly
     livenessProbe:
```

Comment:
GH won't let me comment below. Can you `--enable-endpoint-slices` to the flags. This will use the EPS for updates over endpoints.

Reply:
done